src/coreclr/jit/codegenriscv64.cpp

```diff
@@ -3374,8 +3374,16 @@ void CodeGen::genFloatToIntCast(GenTree* treeNode)
     }
 
     genConsumeOperands(treeNode->AsOp());
+    regNumber tmpReg = rsGetRsvdReg();
 
     GetEmitter()->emitIns_R_R(ins, EA_8BYTE, treeNode->GetRegNum(), op1->GetRegNum());
+
+    // This part emulates the "flush to zero" option because the RISC-V specification does not provide it.
+    GetEmitter()->emitIns_R_I_I(INS_csrrci, EA_8BYTE, tmpReg, 22, 1);
```

Comment:
It seems that you reset `NX`(0) and `DZ`(4) bits here. `FCVT.L.S`, `FCVT.LU.S`, `FCVT.L.D`, `FCVT.LU.D` set only `NV`(5) and `NX`(0) according to spec. Why reset `DZ`?

Reply:
Also would be good to have comment here, that you get `fflags` portion of `fcsr`

Reply:
Added constant mask for 22 and comment for 1